src/popup/pages/Send/index.scss

```diff
@@ -147,6 +147,7 @@ p {
   display: flex;
   flex-direction: row;
   align-items: center;
+  white-space: pre
```

Comment:
:information_source: current structure is like this:

![image](https://user-images.githubusercontent.com/38229038/146059021-8722e1ce-a258-498b-a853-8a6954a5041f.png)

`Available: ` and number are laid out by flex, and the trailing space after the colon gets collapsed.

Reply:
I would rather put `&nbsp;` between those :shrug:

Reply:
I would rather use inline layout for the whole thing :shrug: